src/Http/Controllers/StaticPagesController.php

```diff
@@ -2,33 +2,123 @@
 
 namespace Ourgarage\StaticPages\Http\Controllers;
 
+use Ourgarage\StaticPages\Http\Requests\StaticPageCreateRequest;
 use App\Http\Controllers\Controller;
+use Ourgarage\StaticPages\Models\StaticPage;
+use Notifications;
 
 class StaticPagesController extends Controller
 {
 
-    public function pageList() {
+    public function indexAdmin(StaticPage $staticPages)
+    {
+        $pages = $staticPages->orderBy('updated_at', 'desc')->paginate(20);
 
+        \Title::prepend(trans('dashboard.title.prepend'));
+        \Title::append(trans('static-pages::pages.index.title'));
+
+        if (view()->exists('packages.static-pages.admin.index')) {
+            return view('packages.static-pages.admin.index', ['pages' => $pages]);
+        } else {
+            return view('staticPages::admin.index', ['pages' => $pages]);
+        }
+    }
+
+    public function statusUpdate($id, StaticPage $page)
+    {
+        $page = $page->find($id);
+
+        $page->update([
+            'status' => $page->status == StaticPage::STATUS_ACTIVE ? StaticPage::STATUS_DISABLED : StaticPage::STATUS_ACTIVE,
+        ]);
+
+        Notifications::success(trans('static-pages::pages.notifications.page-status-update'), 'top');
+
+        return redirect()->back();
+    }
+
+    public function createPage()
+    {
+        \Title::prepend(trans('dashboard.title.prepend'));
+        \Title::append(trans('static-pages::pages.create.title'));
+
+        if (view()->exists('packages.static-pages.admin.create-page')) {
+            return view('packages.static-pages.admin.create-page');
+        } else {
+            return view('staticPages::admin.create-page');
+        }
+    }
+
+    public function store(StaticPageCreateRequest $errors, $id = null)
```

Comment:
StaticPageCreateRequest $errors
This is not errors. This is your actual request. Use it instead of `request()`
And name it not `$errors` but `$request`